lib/vm/src/export.rs

```diff
@@ -35,6 +35,8 @@ pub struct ExportFunction {
     pub signature: FunctionType,
     /// The function kind (it defines how it's the signature that provided `address` have)
     pub kind: VMFunctionKind,
+    /// Address of the function call trampoline owned by the same VMContext that owns the VMFunctionBody. May be None when FunctionType == Dynamic or vmctx == nullptr.
```

Comment:
Note: we should remove `or vmctx == nullptr`, it doesn't make sense for trampoline to be None if vmctx is null
```suggestion
    /// Address of the function call trampoline owned by the same VMContext that owns the VMFunctionBody.
    /// May be None when FunctionType == Dynamic.
```

Reply:
If vmctx == nullptr then it must be a host function, and host functions never have trampolines. Is that not right?

Reply:
It got me a bit confused. Perhaps this description is actually better:
```suggestion
    /// Address of the function call trampoline owned by the same VMContext that owns the VMFunctionBody.
    /// May be None when the function is an host-function (FunctionType == Dynamic or vmctx == nullptr)
```